invenio_pidstore/models.py

```diff
@@ -69,6 +69,14 @@ class PIDStatus(object):
     should not be reused for something else.
     """
 
+    attributes = [
```

Comment:
We can discuss more on this topic in https://github.com/inveniosoftware/invenio-pidstore/issues/44